README.md

```diff
@@ -226,6 +330,46 @@ except for predicate1 and predicate2 that will be delegated to the object-based
 }
 ```
 
+#### Probability Generation Fragmentation Strategy
+
+A fragmentation strategy that ignore quads based on a probability.
+`generationProbability` is the probability for the quad to be handled by the `strategy`.
+The `strategy` must be a `FragmentationStrategyStreamAdapter`.
+When `partitionByResourceType` is `true` the stategy will skip resource type of a container instead of working triple by triple.
+For reproducibility a seed *can* be provided using the parameter `randomSeed`.
+
+```json
+{
+  "fragmentationStrategy": { 
+        "@type": "FragmentationStrategyProbabilityQuadHandling",
+        "generationProbability": 33,
+        "partitionByResourceType": true,
+        "randomSeed":0,
+        "strategy": { 
+        "@type": "FragmentationStrategyShape",
```

Comment:
Maybe we can use a simpler sub-strategy in this example. Also do fix indentation.